python-sdk/src/astro/sql/operators/load_file.py

```diff
@@ -180,6 +183,50 @@ def replace_illegal_columns_chars(char: str, database: BaseDatabase) -> str:
 
         return normalize_config
 
+    def get_openlineage_facets(self) -> OpenLineageFacets:
+        """
+        Returns the lineage data
+        """
+        input_files = resolve_file_path_pattern(
+            self.input_file.path,
+            self.input_file.conn_id,
+            normalize_config={},
+            filetype=self.input_file.type.name,
+        )
+
+        input_dataset: list[OpenlineageDataset] = [
+            OpenlineageDataset(
+                namespace=self.input_file.openlineage_dataset_namespace,
+                name=self.input_file.openlineage_dataset_name,
+                facets={
+                    "file_size": self.input_file.size,
+                    "is_pattern": self.input_file.is_pattern(),
+                    "files": [file.path for file in input_files],
+                    "number_of_files": len(input_files),
```

Comment:
Facets should be either standarized facets: https://github.com/OpenLineage/OpenLineage/tree/main/spec/facets

or custom facets that are object deriving from `BaseFacet`.

Reply:
I have created a custom facet here: https://github.com/astronomer/astro-sdk/pull/1025/commits/fd81ed87cbc39ba16cc4d030dafa55a86223f65a